Assets/HoloToolkit-Examples/Input/Scripts/ButtonObjectScaler.cs

```diff
@@ -48,22 +50,28 @@ private void OnButtonPressed(TestButton source)
                 switch (ButtonAction)
                 {
                     case buttonAction.Reset:
-                        Debug.Log(InitialScale);
-                        ObjectToScale.transform.localScale = InitialScale;
-                        break;
+                        {
+                            Debug.Log(InitialScale);
+                            ObjectToScale.transform.localScale = InitialScale;
+                            break;
+                        }
                     case buttonAction.Grow:
-                        ObjectToScale.transform.localScale = new Vector3((ObjectToScale.transform.localScale.x + ScaleIncrement), (ObjectToScale.transform.localScale.y + ScaleIncrement), (ObjectToScale.transform.localScale.z + ScaleIncrement));
-                        break;
-                    case buttonAction.Shrink:
-                        if ((ObjectToScale.transform.localScale.x - ScaleIncrement) < 0.0f)
                         {
-                            ObjectToScale.transform.localScale = new Vector3(0, 0, 0);
+                            ObjectToScale.transform.localScale = new Vector3((ObjectToScale.transform.localScale.x + ScaleIncrement), (ObjectToScale.transform.localScale.y + ScaleIncrement), (ObjectToScale.transform.localScale.z + ScaleIncrement));
+                            break;
                         }
-                        else
+                    case buttonAction.Shrink:
                         {
-                            ObjectToScale.transform.localScale = new Vector3((ObjectToScale.transform.localScale.x - ScaleIncrement), (ObjectToScale.transform.localScale.y - ScaleIncrement), (ObjectToScale.transform.localScale.z - ScaleIncrement));
+                            if ((ObjectToScale.transform.localScale.x - ScaleIncrement) < 0.0f)
+                            {
+                                ObjectToScale.transform.localScale = new Vector3(0, 0, 0);
```

Comment:
`ObjectToScale.transform.localScale = new Vector3(0, 0, 0);` -> ` ObjectToScale.transform.localScale = Vector3.zero`

Reply:
I didn't change this part as well, maybe I'll help you to change the code and upload again.